### DISCUSSION TITLE
Support for fixed output and alternate volume

### DISCUSSION THREAD

**Discussion by monzsca** [punxaphil/custom-sonos-card#488] [Ideas]
Hello, I have a sonos connect with it's volume set to fixed (disables adjusting the volume) and have it connected to the stereo receiver. It works fine, volume shows at full and is not adjustable, as expected. In this case it would be nice to have the option to hide the volume control. Or change the device that volume control controls. In my case it  would be nice to be able to set it to the stereo receiver's volume control (already controllable in home assistant). I know the volume can be set to any sonos device, but any device that has a volume control would be good, if possible. This way it appears like any other sonos device and has volume control. Right now I have a separate card for the receiver's volume.

I suspect not many are using sonos devices with the volume set to fixed, the new app didn't (doesn't?) even have the option to change that setting.

**Comment by punxaphil** [comment]
I'm thinking you can combine these:
```
entities:
  - media_player.my_amp
  - media_player.my_connect
  - ...
playerVolumeEntityId: media_player.my_amp
showNonSonosPlayers: true
```
